assets/js/plugin-check-admin.js

```diff
@@ -35,6 +35,31 @@
 	canRunChecks();
 	pluginsList.addEventListener( 'change', canRunChecks );
 
+	/**
+	 * Saves the user settings.
+	 *
+	 * @since n.e.x.t
+	 */
+	function saveUserSettings() {
+		const selectedCategories = [];
+
+		// Assuming you have a list of category checkboxes, find the selected ones.
+		categoriesList.forEach( function ( checkbox ) {
+			if ( checkbox.checked ) {
+				selectedCategories.push( checkbox.value );
+			}
+		} );
+
+		// Join the selected category slugs with '__' and save it as a user setting.
+		const settingValue = selectedCategories.join( '__' );
```

Comment:
@felixarntz, in the pull request, I opted to use `__` instead of `--` as the latter was causing inconsistencies when viewing the Plugin Check page on various browsers.

Reply:
Maybe a pipe `|` character as a delimiter? Seems more natural

Reply:
@swissspidy `~` also not supported.